app/views/project/editor/editor.pug

```diff
@@ -37,9 +37,9 @@ div.full-size(
 			keybindings="settings.mode",
 			font-size="settings.fontSize",
 			auto-complete="settings.autoComplete",
-			enable-auto-complete-features="true",
+			enable-auto-complete-features="!anonymous"
```

Comment:
Anonymous users may have write permissions, so this should be `permissions.write` (see the read-only attribute on the editor below on line 51).